libcloud/compute/drivers/azure_arm.py

```diff
@@ -808,6 +808,7 @@ def destroy_node(
         ex_destroy_vhd=True,
         ex_poll_qty=10,
         ex_poll_wait=10,
+        ex_destroy_os_disk=False,
```

Comment:
I think it's fine to default it to `True` as long as that change is documented well (including an example of how to set it to `False`) in `docs/upgrade_notes.rst` file.

Reply:
Thanks. I set it to `True` but I still need to update the notes. I understand we want to keep `ex_destroy_vhd` and `ex_destroy_os_disk` separately. The first to delete the OS disk when it is created through a VHD and `ex_destroy_os_disk` when it is a managed disk. Is that correct?

Reply:
Yeah, I think that's correct.

Once you will add the upgrade notes entry, I will go ahead and merge the PR. Thanks.